modules/openthread/platform/diag.c

```diff
@@ -313,3 +339,77 @@ static otError startModCarrier(otInstance *aInstance, uint8_t aArgsLength, char
 }
 
 #endif
+
+static otError processTransmit(otInstance *aInstance, uint8_t aArgsLength, char *aArgs[])
+{
+	otError error = OT_ERROR_NONE;
+
+	if (aArgsLength == 0) {
+		diag_output("transmit will send %" PRId32 " diagnostic messages with %" PRIu32
+			    " ms interval\r\nstatus 0x%02x\r\n",
+			    sTxRequestedCount, sTxPeriod, error);
+	} else if (strcmp(aArgs[0], "stop") == 0) {
+		if (sTransmitMode == kDiagTransmitModeIdle) {
+			error = OT_ERROR_INVALID_STATE;
+		}
+
+		otPlatAlarmMilliStop(aInstance);
+		diag_output("diagnostic message transmission is stopped\r\nstatus 0x%02x\r\n",
+			    error);
+		sTransmitMode = kDiagTransmitModeIdle;
+		otPlatRadioReceive(aInstance, sChannel);
+	} else if (strcmp(aArgs[0], "start") == 0) {
+		if (sTransmitMode != kDiagTransmitModeIdle) {
+			error = OT_ERROR_INVALID_STATE;
+		}
+
+		otPlatAlarmMilliStop(aInstance);
+		sTransmitMode = kDiagTransmitModePackets;
+		sTxCount = sTxRequestedCount;
```

Comment:
is `sTxCount` used? I can see it being assigned only here

Reply:
Pleae see updated implementation.